Makefile

```diff
@@ -4,4 +4,7 @@ files := $(shell find . \( -name '*.h' -or -name '*.hpp' -or -name '*.cpp' \) -n
 format:
 	clang-format -i $(files)
 
+tidy:
+	clang-tidy files
```

Comment:
Why are you adding this if you are deleting .clang-tidy?

Reply:
Deleted